features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/voicemessages/VoiceMessageComposerState.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
```

Comment:
A few added files have 2022 copyright, can you check and put 2023 please? Thanks